src/book/garden-lattice/understanding.md

```diff
@@ -0,0 +1,113 @@
+
+### README
+
+The `README` file is the cornerstone of any repository, providing essential information about the project.
+Its presence signifies a well-documented and accessible project, inviting contributors and users alike to understand and engage with the work.
+A repository thrives when its purpose and usage are communicated through a `README` file, much like a garden benefits from a clear plan and understanding of its layout.
+
+> "A good rule of thumb is to assume that the information contained within the README will be the only documentation your users read.
+> For this reason, your README should include how to install and configure your software, where to find its full documentation, under what license it’s released, how to test it to ensure functionality, and acknowledgments.
+> Furthermore, you should include your quickstart guide (as introduced in Rule 3) in your README." {cite:p}`lee_ten_2018`
+
+For further reading on `README` file practices, see the following resources:
+
+- [The Turing Way: Landing Page - README File](https://book.the-turing-way.org/project-design/project-repo/project-repo-readme.html)
+- [GitHub: About `README`s](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)
+- [Makeareadme.com](https://www.makeareadme.com/)
+
+### CONTRIBUTING
+
+A `CONTRIBUTING` file outlines the guidelines for contributing to the project.
+Its presence fosters a welcoming environment for new contributors, ensuring that they have the necessary information to participate effectively.
+In the same way that a garden flourishes when there's good understanding of roles and how to care for one another, a project grows stronger when contributors are guided by clear and inclusive instructions.
+
+All development procedures which are specific to the project should be outlined within the `CONTRIBUTING` file (for example, testing and linting expectations, release cadence, etc.).
+Consider writing this documentation for each project from the standpoint of an outsider or beginner who may be trying to add a new task, understand the project, or complete an existing task {cite:p}`treude_towards_2024`.
+
+For further reading on `CONTRIBUTING` file practices, see the following resources:
+
+- [GitHub: Setting guidelines for repository contributors](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)
+- [Mozilla: Wrangling Web Contributions: How to Build a CONTRIBUTING.md](https://mozillascience.github.io/working-open-workshop/contributing/)
+
+### CODE_OF_CONDUCT
+
+The `CODE_OF_CONDUCT` (CoC) files set the standards for behavior within the project community.
+Its presence is a commitment to maintaining a respectful and inclusive environment for all participants.
+It also may provide an outline for acceptable participation when it comes to conflicts of interest or similar.
```

Comment:
```suggestion
It also may provide an outline for acceptable participation when it comes to conflicts of interest.
```

Not sure what you mean by "similar" - consider either deleting or expanding with an example.

Reply:
I feel like there might be more here to this but I don't have strong evidence, I'll drop for now. Thanks!